File: examples/thundra/src/lambda.js
import Thundra from "@thundra/core";

const thundra = Thundra({
  apiKey: process.env.THUNDRA_API_KEY,
});

// wrap your lambda function with Thundra
export const handler = thundra(async (event) => {
  return {
    statusCode: 200,
    headers: { "Content-Type": "text/plain" },
    body: `Hello, World! Your request was received`,
  };
});

File: packages/create-sst/bin/presets/starters/go-starter/templates/services/functions/lambda.go
package main

import (
  "github.com/aws/aws-lambda-go/events"
  "github.com/aws/aws-lambda-go/lambda"
)

func Handler(request events.APIGatewayV2HTTPRequest)(events.APIGatewayProxyResponse, error) {
  return events.APIGatewayProxyResponse {
    Body: "Hello, World! Your request was received at " + request.RequestContext.Time + ".",
    StatusCode: 200,
  }, nil
}

func main() {
  lambda.Start(Handler)
}
